readthedocs/settings/base.py

```diff
@@ -333,132 +333,39 @@ def INSTALLED_APPS(self):  # noqa
     SILENCED_SYSTEM_CHECKS = ['fields.W342']
 
     # Logging
-    LOG_FORMAT = "[%(asctime)s] %(levelname)s [%(name)s:%(lineno)s] %(message)s"
+    LOG_FORMAT = '%(name)s[%(process)d]: %(levelname)s %(message)s [%(name)s:%(lineno)s]'
```

Comment:
`name` is duplicated: take a look at this issue https://github.com/rtfd/readthedocs.org/issues/2851